### PR TITLE
Install all dependencies with requirements file

### PR DESCRIPTION
**Pull request by Baschdl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This is available for an Intel based System (e.g. Synology Docker) as well as an
 ## Running docker
 Choose for the fitting docker tag and run the server with the following parameters:
 
-```sudo docker run -p 3000:3000 --mount type=bind,source=/PATH_TO_LOCAL_CONFIG,target=/config --mount type=bind,source=/PATH_TO_LOCAL_LOG,target=/log jomjol/wasserzaehler:DOCKER_TAG```
```
Comment (Baschdl) [inline README.md:27]: If you have to use sudo to execute docker, your user is not member of the `docker` group

**Comment by jomjol** [comment]
I use a own build docker file by purpouse, as I need to run the docker on systems, that have a cheaper processor, that has no AVX2-support. Therefore I compile Keras/Tensorflow on my own with disabled AVX2-support and implement it in a dedicated docker container (jomjol/synology-opencv-tensorflow).
Therefore I cannot move the requirements to a requirements file.

**Comment by Baschdl** [comment]
Wouldn't it make more sense to have one container without AVX2 for synology systems, one with standard tensorflow and one with tensorflow-gpu?
Building tensorflow without AVX2 in the docker build process would also make it more transparent which version is used. Currently this happens as a side effect and isn't obvious from the docker build.

**Comment by jomjol** [comment]
In principle I agree to all, BUT:
1. Compiling Tensorflow without AVX2-support is not an easy thing. I could not get it working within docker (at least not for TF2.1 anymore) and needed to set up a Linux system with quite some handmade changes, which took quite some time to figure out. It also changes from version to version a little bit. Really something for experts :-)
2. The version w/o AVX2 is running on modern processors as well - just slower
3. In this project the calculation power for using the network is really low, so no need for AVX2 or GPU support. Also in an upcoming version I will switch to TF-Lite. This will decrease the calculation time even on a Raspberry B3 to a view 10ms each digit.

**Comment by jomjol** [comment]
Transfer to tflite with no need for AVX2 is done.

**Pull request closed without merging**